app/widgets/Flanker/index.js

```diff
@@ -0,0 +1,131 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { WebView, View, Text, StyleSheet } from 'react-native';
+import RNFetchBlob from 'rn-fetch-blob';
+import { colors } from '../../theme';
+
+/**
+ * This component tries to download the flanker images and convert them to
+ * base64 strings. The images are cached using rn-fetch-blob for future offline
+ * use.
+ */
+
+const styles = StyleSheet.create({
+  errorContainer: {
+    flex: 1,
+    backgroundColor: 'white',
+    justifyContent: 'center',
+    alignItems: 'center',
+    padding: 30,
+  },
+  errorText: {
+    color: colors.grey,
+  },
+});
+
+const requestImage = url => RNFetchBlob
+  .config({
+    fileCache: true,
+  }).fetch('GET', url)
+  .then((res) => {
+    const { status } = res.info();
+    if (status !== 200) {
+      throw new Error('Status not 200');
+    }
+    return res.base64();
+  });
+
+export class Flanker extends React.Component {
+  constructor() {
+    super();
+    this.state = { loadingState: 'idle' };
+  }
+
+  componentDidMount() {
+    const { config } = this.props;
+    this.setState({ loadingState: 'downloading' });
+    Promise.all([
+      requestImage(config.leftImage),
+      requestImage(config.leftImageButton),
+      requestImage(config.rightImage),
+      requestImage(config.rightImageButton),
+    ]).then((base64Images) => {
+      this.setState({
+        loadingState: 'ready',
+        images: base64Images,
+      });
+    }).catch(() => {
+      // this.setState({ loadingState: 'downloadError' });
+      this.setState({ loadingState: 'ready' });
+    });
+  }
+
+  render() {
+    const { onChange, config, isCurrent } = this.props;
+    const { loadingState, images } = this.state;
+    if (loadingState === 'ready' && isCurrent) {
+      // const injectConfig = `
+      //   window.CONFIG = {
+      //     leftImage: 'data:image/png;base64, ${images[0]}',
+      //     leftImageButton: 'data:image/png;base64, ${images[1]}',
+      //     rightImage: 'data:image/png;base64, ${images[2]}',
+      //     rightImageButton: 'data:image/png;base64, ${images[3]}',
+      //     experimentLength: ${config.experimentLength},
+      //   };
+      //   start();
+      // `;
```

Comment:
Waiting for https://github.com/ChildMindInstitute/mindlogger-app-backend/issues/131 to resolved -- hardcoding config vals until then